pipes/reddit-auto-posts/app/api/pipeline/route.ts

```
@@ -0,0 +1,262 @@
+    const oneMinuteAgo = new Date(now.getTime() - interval);
+
+    const screenData = await pipe.queryScreenpipe({
+      startTime: oneMinuteAgo.toISOString(),
+      endTime: now.toISOString(),
+      windowName: windowName,
+      limit: pageSize,
+      contentType: contentType,
+    });
+
+    let logEntry: DailyLog | undefined;
+    if (screenData && screenData.data && screenData.data.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)